### ISSUE TITLE
QFAPI regression: Registries don't seem to be getting un-frozen

### ISSUE THREAD

**Issue by LemmaEOF** [QuiltMC/quilted-fabric-api#108]
[crash report](https://mclo.gs/6wOmUIp)

It looks like QFAPI+QSL 7.0.5+0.84.0-1.20.1 from Modrinth, on top of Quilt Loader 0.19.2, is failing to un-freeze registries for client and dedicated-server init. This is resulting in any Fabric mods that perform the recommended-against but still Technically Legal action of registering to registries during client init to crash during the launch process. I might be able to PR a fix myself later, but I'll have to see based on time/availability.

**Comment by rinOfTheStars** [comment]
Can't seem to actually replicate this so far! Made a testmod that registers a SoundEvent in a client mod init, and it works fine—other clientside code is able to see it.

**Comment by LemmaEOF** [comment]
yeah, it seems like this might be something Specifically Fucked with the blanketcon pack, so we'll have to do more intense testing later
